api/api/api_exception.py

```diff
@@ -18,10 +18,16 @@ def __init__(self, code: int, details: str = None):
         """
         self.code = code
         self.details = details
+        # show relative paths in exceptions
         self.exceptions = {
-            2000: f'Some parameters are not expected in the configuration file ({CONFIG_FILE_PATH})',
+            2000: 'Some parameters are not expected in the configuration file '
+                  f'(WAZUH_PATH/{to_relative_path(CONFIG_FILE_PATH)})',
             2001: 'Error creating or reading secrets file. Please, ensure '
-                  f'there is enough disk space and permission to write in {SECURITY_PATH}'
+                  'there is enough disk space and permission to write in '
+                  f'WAZUH_PATH/{to_relative_path(SECURITY_PATH)}',
+            2002: 'Error migrating configuration from old API version. '
+                  'Default configuration will be applied',
+            2003: 'Error loading SSL/TLS certificates'

```

Comment:
Same here. This error does not answer the question of the user: why cannot be loaded? We need to give more help and assist on how to fix the problem.

Reply:
Done!